hazelcast/src/main/java/com/hazelcast/config/MapConfig.java

```diff
@@ -763,6 +765,26 @@ public MapConfig setEventJournalConfig(@Nonnull EventJournalConfig eventJournalC
         return this;
     }
 
+    /**
+     * Gets the {@code TieredStoreConfig} for this {@code MapConfig}
+     *
+     * @return tiered-store config
+     */
+    public TieredStoreConfig getTieredStoreConfig() {
+        return tieredStoreConfig;
+    }
+
+    /**
+     * Sets the {@code TieredStoreConfig} for this {@code MapConfig}
+     *
+     * @param tieredStoreConfig tiered-store config
+     * @return this {@code MapConfig} instance
+     */
+    public MapConfig setTieredStoreConfig(TieredStoreConfig tieredStoreConfig) {
```

Comment:
if tiered-store is an enterprise feature, could you please add a validator when it is enabled in OS? Validators can be add here: `com.hazelcast.internal.config.ConfigValidator`

Reply:
Thanks for the suggestion, yes 'only-EE' config validator for tiered-store is added now.